### PR TITLE
Task-40914: Notify the dlp group members when a document is detected.

### PR DESCRIPTION
**Pull request by ahmadmido**
- Before this fix, all admin members and groups which have permissions receive notifications when DLP items are detected.
- This fix will assure that only members of '/platform/dlp' group receive DLP notifications.

### CODE REVIEW

File: component/notification/src/main/java/org/exoplatform/social/notification/plugin/DlpAdminDetectedItemPlugin.java [hunk 1]
```diff
@@ -84,38 +85,20 @@ public boolean isValid(NotificationContext ctx) {
...
+        // Get DLP members group and super user
+        List<String> members = new ArrayList<>();
+        try {
+            ListAccess<User> adminMembersAccess = organizationService.getUserHandler().findUsersByGroupId(DLP_GROUP);
+            int totalAdminGroupMembersSize = adminMembersAccess.getSize();
+            User[] users = adminMembersAccess.load(0, totalAdminGroupMembersSize);
+            members = Arrays.stream(users)
+                            .map(User::getUserName)
+                            .collect(Collectors.toList());
+            members.add(userACL.getSuperUser());
```
Comment (rdenarie) [inline component/notification/src/main/java/org/exoplatform/social/notification/plugin/DlpAdminDetectedItemPlugin.java]: Why do we add super user ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: component/notification/src/main/java/org/exoplatform/social/notification/plugin/DlpAdminDetectedItemPlugin.java (hunk 1 above)
Comment (ahmadmido) [reply component/notification/src/main/java/org/exoplatform/social/notification/plugin/DlpAdminDetectedItemPlugin.java]: superUser doit recevoir aussi notification puisqu'il a la DLP page par defaut.

Comment (rdenarie) [reply component/notification/src/main/java/org/exoplatform/social/notification/plugin/DlpAdminDetectedItemPlugin.java]: For me, the super user should not receive the notification if he is not in the dlp group.
Please ask to PO for clarification

**Review by rdenarie — DISMISSED** [review]

**Review by rdenarie — DISMISSED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Review by rdenarie — APPROVED** [review]

**Pull request merged**